src/kudu/consensus/consensus_meta.h

```diff
@@ -225,6 +222,9 @@ class ConsensusMetadata : public RefCountedThreadSafe<ConsensusMetadata> {
   // Return the specified config.
   const RaftConfigPB& GetConfig(RaftConfigState type) const;
 
+  // Helper function to extend previous_vote_history_
+  void populate_previous_vote_history(const PreviousVotePB& prev_vote);
```

Comment:
based on naming convention, should this not be PopulatePreviousVoteHistory?
sets and simple gets(properties) should be fine the way it is.

Reply:
I am confused about this. I followed the convention of the calling function. There are functions like: `GetConfigOpIdIndex`. I can change it to whatever you like.